aptible-resource.gemspec

```diff
@@ -21,11 +21,11 @@ Gem::Specification.new do |spec|
 
   # HyperResource dependencies
   spec.add_dependency 'uri_template', '>= 0.5.2'
-  spec.add_dependency 'faraday',      '~> 0.9.2'
+  spec.add_dependency 'faraday',      '~> 0.11.0'
   spec.add_dependency 'json'
 
   spec.add_dependency 'fridge'
-  spec.add_dependency 'activesupport', '>= 4.0', '< 6.0'
+  spec.add_dependency 'activesupport', '>= 5.0', '< 6.0'
```

Comment:
Can you clarify why we need to drop support for activesupport 4.x here? The current version specifier allows using 4.x or 5.x, which allows more flexibility for end-users (depending on either their Rails or Ruby version). Unless there's a compelling reason, we shouldn't drop support for activesupport 4.x.